lib/config.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+function configure () {
+  return Object.freeze({
+    "server": Object.freeze({
+      "hostname": process.env.SERVER_HOSTNAME || "localhost",
+      "port": process.env.SERVER_PORT || 8001
+    }),
+
+    "logger": Object.freeze({
+      "level": process.env.LOGGING_LEVEL || "info"
+    }),
+
+    "statsd": Object.freeze({
+      "host": process.env.STATSD_HOST || "localhost",
+      "port": process.env.STATSD_PORT || 8125
+    }),
+
+    "openhim": {
```

Comment:
@Zooloo2014 Perhaps this object should be frozen to prevent accidentally reassigning the original config values. In the `index..js` file where the urn is added perhaps make a copy of this object and manipulate that instead eg:

```js
const apiConf = Object.assign({urn: mediatorConfig.urn}, appConfig.openhim)
```

Reply:
This object is modified by the app 
https://github.com/jembi/facility-cache/blob/84e50bdfa4244ee5353e5a2af794fa7fe161898e/lib/index.js#L57